sos/plugins/__init__.py

```diff
@@ -744,6 +744,11 @@ def _copy_dir(self, srcpath):
                 path = os.path.join(srcpath, name)
                 self._do_copy_path(path)
         except OSError as e:
+            if e.errno == errno.EPERM or errno.EACCES:
+                msg = "Permission denied likely due to Linux Security " \
+                       + "Modules (LSM) restrictions"
+                self._log_warn("_copy_dir: '%s' %s" % (srcpath, msg))
+                return
```

Comment:
I am OK with catching (not re-raising) these exceptions, just having 2 comments:

- afaik copying directory should not hit `EPERM` that applies to operations not permitted (user cant kill `root's` process or similar), but as some applications might mis-use this error code instead of `EACCES`, and since both are some permission errors, I am OK with having both caughts
- I would strip away the "likely due to LMS" and just mention "Permission denied:" on given `srcpath` and probably also on the error details (that can give more data than this "guess")

Reply:
Done